en/ExternalFiles.md

```diff
@@ -16,6 +16,32 @@ These are by default set to values that probably make sense for your operating s
 
 To change the external viewer settings, go to **Options → Preferences → External programs**.
 
+## Pushing to external viewer application
+
+JabRef allows you to push any entries in your main window to an external viewer through the push-to-external application feature. You would need to first select the entries in your entry table that you would like to export before using the feature. Once you have done so, go to the tools submenu and click on the push-to-external application button to the left of the **Generate BibTeX keys** button. By default the external viewer used to push exports is TeXstudio.
+
+On MacOS:
+![Push to External MacOS](images/Push-External-Button-MacOS.png)
+
+On Windows:
```

Comment:
Could you add two spaces (`  `) after "On Windows:"? This leads to a hard line break. (Please also at "On Mac OS:")